runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -476,6 +477,9 @@
          */
         @Default
         int prometheusMetricsPort = MetricsUtils.NO_METRICS_PORT;
+
+        @Default
+        String codecType = Codec.Type.ZSTD.toString();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4705140)